Objects/bytesobject.c

```diff
@@ -1438,7 +1438,7 @@ bytes_concat(PyObject *a, PyObject *b)
     if (PyObject_GetBuffer(a, &va, PyBUF_SIMPLE) != 0 ||
         PyObject_GetBuffer(b, &vb, PyBUF_SIMPLE) != 0) {
         PyErr_Format(PyExc_TypeError, "can't concat %.100s to %.100s",
-                     Py_TYPE(a)->tp_name, Py_TYPE(b)->tp_name);
+                     Py_TYPE(b)->tp_name, Py_TYPE(a)->tp_name);
```

Comment:
I'll do this in separate PR, because this is larger change that shouldn't be backported. The same error message is generated in yet one place (with correct order of arguments). Maybe will make the message being more similar to other concatenate messages.